osu.Game.Rulesets.Osu/Difficulty/OsuPerformanceCalculator.cs

```diff
@@ -32,14 +36,23 @@ public OsuPerformanceCalculator()
         protected override PerformanceAttributes CreatePerformanceAttributes(ScoreInfo score, DifficultyAttributes attributes)
         {
             var osuAttributes = (OsuDifficultyAttributes)attributes;
+            useClassicSlider = score.Mods.Any(h => h is OsuModClassic cl && cl.NoSliderHeadAccuracy.Value);
```

Comment:
`usingClassicSliderMechanics` sounds better to me

Reply:
resolved by #27063 creating variable usingClassicSliderAccuracy